File: blood-bank-be/bloodBank/src/main/java/ftn/uns/ac/rs/bloodbank/appointment/dto/MedicalStaffResponse.java
package ftn.uns.ac.rs.bloodbank.appointment.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.NonNull;

import java.util.UUID;
@Data
@NoArgsConstructor
@AllArgsConstructor
public class MedicalStaffResponse {
    @NonNull
    private UUID id;

    @NonNull
    private String username;

    @NonNull
    private String name;

    @NonNull
    private String surname;
    @NonNull
    private String email;
}

File: blood-bank-fe/src/app/model/Responses/MedicalStaffResponse.ts
export interface IMedicalStaffResponse{
  id?: string,
  username?: string,
  name?: string,
  surname?: string,
  email?: string,
}
export class MedicalStaffResponse implements IMedicalStaffResponse {
  email?: string;
  id?: string;
  name?: string;
  surname?: string;
  username?: string;

  constructor() {
  }
}
